src/workflows-app/SubmissionDetails.js

```diff
@@ -255,6 +277,22 @@ export const BaseSubmissionDetails = ({ name, namespace, workspace, submissionId
             ]),
           ]
         ),
+        logsModal.showLog &&
+          h(LogViewer, {
+            modalTitle: logsModal.modalTitle,
+            logs: logsModal.logsArray,
+            onDismiss: () => {
+              setLogsModal({ modalTitle: logsModal.modalTitle, logsArray: logsModal.logsArray, showLog: false });
+              captureEvent(Events.workflowsAppCloseLogViewer);
+            },
+          }),
+        taskDataModal.showData &&
+          h(InputOutputModal, {
+            title: taskDataModal.taskDataTitle,
+            jsonData: taskDataModal.taskJson,
```

Comment:
Can we update the modal to include the task name, as it appears on the submission input and output tabs? It's common for many tasks to use inputs or outputs of the same name (like an input called `dockerImage`), so for users to understand what they're looking at, it's important to include which task the input or output relates to. We don't know this on the task input/output modal because those values are already associated with a single task.